apps/web/src/routes/AllView/AllView.jsx

```diff
@@ -3,8 +3,8 @@ import { useSelector, useDispatch } from 'react-redux'
 import { useTranslation } from 'react-i18next'
 import { useLocation, useNavigate, useParams } from 'react-router-dom'
 import { capitalize } from 'lodash'
-import { widgetUrl, widgetTitleResolver, widgetTypeResolver, isValidHomeWidget, humanReadableTypes, widgetIsValidChild } from 'util/contextWidgets'
-import { addQuerystringToPath, baseUrl } from 'util/navigation'
+import { ContextWidgetPresenter, widgetIsValidChild, humanReadableTypes, isValidHomeWidget, widgetTypeInferrer } from '@hylo/shared/src/ContextWidgetPresenter'
```

Comment:
isValidHomeWidget, inferredType, and isValidChild seem to be like they should be resolved in the Presenter